lib/rest-adapter.js

```diff
@@ -586,7 +609,7 @@ function RestMethod(restClass, sharedMethod) {
   this.notes = sharedMethod.notes;
   this.documented = sharedMethod.documented;
 
-  var methodRoutes = getRoutes(sharedMethod);
+  var methodRoutes = getRoutes(sharedMethod, options);
```

Comment:
I am proposing to read `options` from `restClass` argument (or `this.restClass`), thus keeping the number of arguments of `RestMethod` smaller.